docs/api/mount/hasAttribute.md

```diff
@@ -1,17 +1,14 @@
-# hasAttribute(attribute, value)
```

Comment:
Sorry, I meant to keep the hasAttribute docs as they were. At the moment details on value are being removed:

For example, `hasAttribute(attribute)` should be `hasAttribute(attribute, value)` and `Check if wrapper DOM node has specified attribute` should be `Check if wrapper DOM node has attribute matching value`